### ISSUE TITLE
whitespace rule with check-typecast incompatible with TS 2.4's dynamic import()

### ISSUE THREAD

**Issue by Kovensky** [palantir/tslint#2920]
### Bug Report

- __TSLint version__: 5.4.3
- __TypeScript version__: 2.4.0
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
import('./foo')
.then(foo => { foo.setup() })
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "whitespace": [true, "check-typecast"]
  }
}
```

#### Actual behavior

"missing whitespace" at the opening parenthesis of `import(`.

#### Expected behavior

No error.
